R/colby_constructors.R

```diff
@@ -1987,16 +1987,17 @@ list_wrap_df <- function(f) {
 #'
 #' Every layout must start with a basic table.
 #'
-#' @export
 #' @inheritParams constr_args
 #' @param show_colcounts logical(1). Should column counts be displayed in the
 #'   resulting table when this layout is applied to data
 #' @param colcount_format character(1). Format for use when displaying the
-#' column counts. Must be 1d, or 2d where one component is a percent. See
-#' details.
+#'   column counts. Must be 1d, or 2d where one component is a percent. See
+#'   details.
+#' @param top_level_section_div character(1). If assigned to a single character,
+#'   the larger sections of the table will be divided by a line made of that character.
```

Comment:
Can you be a bit more specific about what a "larger section" is?

Reply:
```suggestion
#'   the first (top level) split or division of the table will be highlighted by a line made of that character.
```